### DISCUSSION TITLE
Possible typo in client scope mapper help text for User Attribute

### DISCUSSION THREAD

**Discussion by danthemango** [keycloak/keycloak#33545] [Q&A] [answered]
The help text for a User Attribute mapper says that it is used to "Map a custom user attribute **to a to a** SAML attribute." on [this line of code](https://github.com/keycloak/keycloak/blob/edb05421ac13819b903ac6e7b5b264067e575d09/services/src/main/java/org/keycloak/protocol/saml/mappers/UserAttributeStatementMapper.java#L84), am I correct in assuming it should say "Map a custom user attribute to a SAML attribute."?

**Comment by codespearhead** [comment, marked as answer]
Yes, you're correct.

I've just created a PR [1] to address this.

Could you mark this comment as the accepted answer [2] for this discussion, so it no longer appears as unanswered in the GitHub discussions tab?

[1] https://github.com/keycloak/keycloak/pull/33548
[2] https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer

**Reply by danthemango** [reply]
Sure thing
